.github/workflows/android_build.yaml

```diff
@@ -16,4 +17,28 @@ jobs:
           java-version: 11
 
       - name: Run tests
-        run: ./gradlew test
\ No newline at end of file
+        run: ./gradlew test
+
+  instrumentation_tests:
+    name: Instrumentation Tests
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        api-level: [26]
+        arch: [x86, x86_64]
```

Comment:
Is there a particular reason to test on x86? I'm not super familiar with CI testing and all but I just figure the more modern `x86_64` is plenty good for us. Let me know if you had a particular use case for the different architectures in the matrix.

Reply:
Theres not particular reason for using `x86`, I just thought that since we support both `x86` and `x86_64` we can test both.